src/content/docs/auto-telemetry-pixie/install-auto-telemetry-pixie.mdx

```diff
@@ -142,7 +142,7 @@ If you arrived in the guided installation process by following a link from Pixie
     id=""
```

Comment:
Do you have to specifically add an ID label? I'm not sure what the docs does if this is left blank, but I've always added a text label just to be sure. Hope this helps!